V3-Upgrade-Guide.md

```diff
@@ -37,3 +37,23 @@ For historic reasons the command-line arguments of both applications were differ
 In version 3, the arguments are aligned and deprecated arguments have been removed.
 Please see (insert link here) for the current documentation on the cli arguments.
 -->
+
+### Timeout 
+
+The `timeout` option on the request builder as well as the http-client was removed. 
+If you want to set a timeout for a request use the new timeout middleware:
+
+```ts
+ executeHttpRequest(myDestination, {
+          method: 'get',          
+          middleware: [timeout()]
```

Comment:
Without reading the rest of the PR (which the user won't do) I'm not sure if there is something missing here. Do I pass the actual timeout value to the function? In what unit?

Reply:
apparently the answer is 'yes, in milliseconds'. should be documented here imo.

Reply:
it in the JS docs